simpletransformers/t5/t5_model.py

```diff
@@ -104,7 +111,10 @@ def __init__(
             self.args.fp16 = False
 
         self.args.model_type = "T5"
-        self.args.model_name = model_name
+        if model_name==None:
+            self.args.model_name = "T5fromScratch"
+        else:
+            self.args.model_name = model_name
```

Comment:
```suggestion
        if model_name is None:
            self.args.model_name = "T5_from_scratch"
        else:
            self.args.model_name = model_name
```

Let's make it like this then? Changed to underscores for consistency.

Reply:
Yes, the name was just a proposal :)